04 DisplayData/src/components/members/membersPage.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import memberEntity from '../../api/memberEntity';
-import MemberAPI from '../../api/memberAPI';
+import memberAPI from '../../api/memberAPI';
```

Comment:
Great! Option 2 is better for me too.

Do I change all export default to export in this step?